nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/proxy/HelmProxyRecipe.groovy

```diff
@@ -106,11 +108,6 @@ class HelmProxyRecipe
           .create())
     }
 
-    builder.route(new Route.Builder()
-        .matcher(BrowseUnsupportedHandler.MATCHER)
```

Comment:
minor: the `BrowseUnsupportedHandler` import at the top of this file is now unused and can be removed